src/pydata_sphinx_theme/__init__.py

```diff
@@ -1166,6 +1166,9 @@ def copy_logo_images(app: Sphinx, exception=None) -> None:
         path_image = logo.get(f"image_{kind}")
         if not path_image or isurl(path_image):
             continue
+        if (staticdir / Path(path_image).name).exists():
+            # file already exists in static dir
```

Comment:
```suggestion
            # file already exists in static dir
            # e.g., because a theme has already bundled it there
```

Reply:
Slight edit, but general idea is the same:
```
            # file already exists in static dir e.g. because a theme has
            # bundled the logo and installed it there
```